src/modules/fancyzones/editor/FancyZonesEditor/GridEditor.xaml.cs

```diff
@@ -137,80 +137,94 @@ private void ExtendRangeToHaveEvenCellEdges()
             // extend each edge of the [(_startCol, _startRow) - (_endCol, _endRow)] range based on merged cells until you have 4 straight edges with no "straddling cells"
```

Comment:
Function name ExtendOverhangingPartsToMinimalProperRectangle

Reply:
This name is arguable but let's just leave it as-is.